sum_types_generator/lib/src/templates.dart

```diff
@@ -141,6 +141,35 @@ String function({
         ";",
     ].join("\n");
 
+String expressionFunction({
+  required String type,
+  required String name,
+  required Iterable<String> body,
+  bool isStatic = false,
+  Iterable<TypeParamSpec> typeParams = const [],
+  Iterable<String> posParams = const [],
+  Iterable<String> namedParams = const [],
+}) =>
+    [
+      if (isStatic) "static ",
+      type,
+      " ",
+      name,
+      parametrize(typeParams),
+      "(",
+      ...posParams.map(appendComma),
+      if (namedParams.isNotEmpty) ...[
+        "{",
+        ...namedParams.map(appendComma),
+        "}",
+      ],
+      ") =>",
+      ...[
+        body.join("\n"),
+      ],
```

Comment:
Here you are constructing a list with a single element and then including all its elements (the one) into the resulting list. You can completely drop this extra list, just directly include that single element:

```dart
body.join("\n"),
```

(without surrounding it with `...[]`)